third_party/maya/lib/usdMaya/usdImport.cpp

```diff
@@ -109,18 +111,28 @@ MStatus usdImport::doIt(const MArgList & args)
         MString tmpVal;
         argData.getFlagArgument("file", 0, tmpVal);
 
-        // resolve the path into an absolute path
-        MFileObject absoluteFile;
-        absoluteFile.setRawFullName(tmpVal);
-        absoluteFile.setRawFullName( absoluteFile.resolvedFullName() ); // Make sure an absolute path
-
-        if (!absoluteFile.exists()) {
-            MGlobal::displayError("File does not exist.  Exiting.");
+        // First resolve using maya's resolver, to expand any environment
+        // variables, or any maya-specific tokens
+        MFileObject mayaFile;
+        mayaFile.setRawFullName(tmpVal);
+
+        // Use the usd resolver - this should also have the side effect of making
+        // absolute paths
+        std::string expandedPath = mayaFile.expandedFullName().asChar();
+        ArResolver& resolver = ArGetResolver();
+        resolver.ConfigureResolverForAsset(expandedPath);
```

Comment:
I don't think we should reconfigure the resolver here, since that's a stateful change that potentially affects all subsequent consumers of ArResolver for path resolution. Can you describe what issue you were running into here?